Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -182,7 +182,12 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     // Log share initiated event
     [self userCompletedAction:BNCShareInitiatedEvent];
     UIActivityItemProvider *itemProvider = [self getBranchActivityItemWithLinkProperties:linkProperties];
-    NSMutableArray *items = [NSMutableArray arrayWithObject:itemProvider];
+    NSMutableArray *items;
+    if (itemProvider) {
```

Comment:
Couldn`t reproduce this with the test bed application. Setting a `itemProvider = nil` creates the exact same error log as mentioned in the ticket. Looking at the code there is no chance for this since it is returning the `UIActivityViewController`